readme.md

```diff
@@ -552,6 +552,7 @@ Just type [`node.cool`](https://node.cool) to go here ✨
 - [Stylecow](https://github.com/stylecow/stylecow) - Parse, manipulate and convert modern CSS to make it compatible with all browsers. Extensible with plugins.
 - [js-yaml](https://github.com/nodeca/js-yaml) - Very fast YAML parser.
 - [excel-stream](https://github.com/dominictarr/excel-stream) - Streaming Excel spreadsheet to JSON parser.
+- [fast xml parser](https://github.com/NaturalIntelligence/fast-xml-parser) - Very fast XML to Js Obj/JSON or Nimn and JSON/Js obj to XML converter. It can validate XML syntax and can work from CLI as well.
```

Comment:
The name should be identical to the npm package name. The description also needs work. Try to be as short as possible.

Reply:
Thanks for your review. Done :+1: